File: count-factor/javascript/countfactors.js
const countFactor = (A) => {
    let counter = 0;
    for (let i = 1; i * i <= A; i++) {
        if(A%i == 0){
            if(A/i == i){
                counter = counter+1;
            }else{
                counter = counter+2;
            }
        }
    }

    return counter;
}

console.log(countFactor(10));

File: count-factor/java/CountFactors.java
public class CountFactors {
    static int countFactor(int A) {
        int counter = 0;
        for(int i=1;i*i<=A;i++){
            if(A%i==0){
                if(A/i == i){
                    counter ++;
                }else{
                    counter = counter+2;
                }
            }
        }
        return counter;
    }

    public static void main(String[] args) {
        System.out.println(countFactor(1));
    }
}